contracts/exchange/contracts/src/MixinSignatureValidator.sol

```diff
@@ -210,6 +231,11 @@ contract MixinSignatureValidator is
         );
     }
 
+    function _getCurrentContextAddress()
```

Comment:
Same comment as above - how is this used?

Reply:
I think it probably makes sense to just inherit `MixinTransactions` whenever they are used.